01-messaging.md

```diff
@@ -255,6 +255,7 @@ The receiving node:
     - MUST ignore the bit.
   - upon receiving unknown _even_ feature bits that are non-zero:
     - MUST fail the connection.
+  - MUST validate all known, transitive feature dependencies.
```

Comment:
MUST validate doesn't mean anything - should I disconnect? Should I ignore the message? Should I....

Reply:
oops, forgot the most important part, thanks! both have been updated